src/kdc/do_as_req.c

```diff
@@ -841,6 +841,8 @@ prepare_error_as(struct kdc_request_state *rstate, krb5_kdc_req *request,
     kdc_realm_t *kdc_active_realm = rstate->realm_data;
     size_t count;
 
+    memset(&errpkt, 0, sizeof(krb5_error));
```

Comment:
Right, it should nominally be set to KV5M_ERROR anyway, not to 0 (although it doesn't matter in practice).